system/web/Renderer.cfc

```diff
@@ -946,7 +948,7 @@ component
 	/**
 	 * Checks if implicit views are turned on and if so, calculate view according to event.
 	 */
-	private function implicitViewChecks(){
+	private function implicitViewChecks( required string view ){
```

Comment:
I have an issue with the changes because then implicit views will be broken because nobody is setting the implicit view anymore since you change the `setView()` methods.  Did you test this?

Reply:
Also, I see the point of the pre-emptive implicit check being too pre-emptive. However, I do see this introducing a regression.

Reply:
It passed all the tests and worked in our API. If there is another test case that is missing can you add it to the test file?